pajbot/bot.py

```diff
@@ -22,7 +22,7 @@
 from pajbot.apiwrappers.twitch.helix import TwitchHelixAPI
 from pajbot.apiwrappers.twitch.id import TwitchIDAPI
 from pajbot.apiwrappers.twitch.tmi import TwitchTMIAPI
-from pajbot.constants import VERSION
+from pajbot.constants import VERSION, CHARACTER_LIMIT
```

Comment:
```suggestion
from pajbot.constants import CHARACTER_LIMIT, VERSION
```

Imports are ordered alphabetically